packages/mdc-slider/_mixins.scss

```diff
@@ -446,11 +434,17 @@
 @mixin rail-color_($color, $opacity: variables.$baseline-rail-opacity, $query: feature-targeting-functions.all()) {
   $feat-color: feature-targeting-functions.create-target($query, color);
 
-  .mdc-slider__track-container::after {
```

Comment:
Revert.

Or sync your CL to head and re-export.

Reply:
Done